api/src/wfl/api/spec.clj

```diff
@@ -10,6 +10,7 @@
 
 ;; shared
 (s/def ::base_file_name string?)
+(s/def ::common map?)
```

Comment:
Revert this file too please - @tbl3rd is working on this

Reply:
Reverted.